README.md

```diff
@@ -1 +1,7 @@
-# Norman Utils External Package
\ No newline at end of file
+# Norman Utils Overview
+
+Norman Utils is a shared library of helper utilities designed for SDK developers working within the Norman ecosystem.
+
+These utilities are used across both the Norman SDK and internal Norman services. They provide common functionality that helps developers extend capabilities, debug issues and integrate advanced features more efficiently.
+
+Comprehensive documentation is available on our website: https://sdk.norman-ai.com/utils
```

Comment:
For the full reference and detailed instructions, please visit our Core SDK documentation at https://sdk.norman-ai.com/utils.